core/templates/dev/head/app.js

```diff
@@ -471,7 +471,8 @@ oppia.factory('oppiaDatetimeFormatter', ['$filter', function($filter) {
       var date = new Date(millisSinceEpoch);
       if (date.toLocaleDateString() === new Date().toLocaleDateString()) {
         // The replace function removes 'seconds' from the time returned.
-        return date.toLocaleTimeString().replace(/:\d\d /, ' ');
+        return date.toLocaleTimeString('en-US',
```

Comment:
Hm. I'm OK with this, but just wondering, why en-US instead of the locale convention?